engine/apps/api/views/alert_group.py

```diff
@@ -54,101 +58,62 @@ def get_user_queryset(request):
     return User.objects.filter(organization=request.user.organization).distinct()
 
 
-class AlertGroupFilterBackend(filters.DjangoFilterBackend):
```

Comment:
Removing this as it doesn't work (the number of SQL queries is the same without this code)